katacomb/katacomb/uv_facade.py

```diff
@@ -401,6 +401,26 @@ def Desc(self):
     def List(self):
         return self.uv.List
 
+    @property
```

Comment:
Properties are generally preferred for things that behave like attributes e.g. are quick to compute and won't throw exceptions. This looks like it might be more suitable as a regular function?

Reply:
OK. Done.